packages/app/src/plugins/webview-preload.js

```diff
@@ -33,24 +14,15 @@ if (micromatch.isMatch(window.location.origin, originsAllowed) || protocolsAllow
     return p;
   };
 
-  const sendSubscribeToProxy = (channel) => {
-    const o = new Observable(obs => {
-      ipcRenderer.on(`bx-api-subscribe-response-${channel}`, (_, result) => {
-        obs.next(result);
-      });
-    });
+  const addListenerToChannel = (channel, listener) => {
+    ipcRenderer.on(`bx-api-subscribe-response-${channel}`, listener);
     setTimeout(() => {
       ipcRenderer.invoke('get-worker-contents-id')
           .then(workerWebContentsId => ipcRenderer.sendTo(workerWebContentsId, 'bx-api-subscribe', channel));
     }, 1);
```

Comment:
The unsubscriber mentionned above could be added here:
```ts
return () => ipcRenderer.off(`bx-api-subscribe-response-${channel}`, listener);
```

Reply:
done